mackup/main.py

```diff
@@ -1,7 +1,7 @@
 """Mackup.
 
 Keep your application settings in sync.
-Copyright (C) 2013-2015 Laurent Raufaste <http://glop.org/>
+Copyright (C) 2013-2018 Laurent Raufaste <http://glop.org/>
```

Comment:
could be updated to 2021 nowadays

Reply:
https://github.com/lra/mackup/pull/1023/commits/b88f6caf0cb5c7d46dae5390b4a623edaa4296a4